matter/wifi/wf200/efr_spi.c

```diff
@@ -72,78 +71,86 @@ static uint32_t dummy_rx_data;
 static uint32_t dummy_tx_data;
 static bool spi_enabled = false;
 
-uint8_t wirq_irq_nb = SL_WFX_HOST_PINOUT_SPI_IRQ; // SL_WFX_HOST_PINOUT_SPI_WIRQ_PIN;
+#if defined(EFR32MG12)
+uint8_t wirq_irq_nb = SL_WFX_HOST_PINOUT_SPI_IRQ;
```

Comment:
wirq_irq_nb for MG12 is different when compared to MG24. This configuration is missed out